sopel/trigger.py

```diff
@@ -201,10 +206,10 @@ def __init__(
 
         # Parse CTCP into a form consistent with IRCv3 intents
         if self.event == 'PRIVMSG' or self.event == 'NOTICE':
-            intent_match = PreTrigger.intent_regex.match(self.args[-1])
-            if intent_match:
-                intent, message = intent_match.groups()
-                self.tags['intent'] = intent
+            ctcp_match = PreTrigger.ctcp_regex.match(self.args[-1])
+            if ctcp_match:
```

Comment:
Is this a good time to make this check a bit more explicit?

```suggestion
            if ctcp_match is not None:
```